app/src/main/java/org/fossasia/openevent/general/LoginActivity.java

```diff
@@ -92,6 +106,34 @@ public void loginUser(String title, String body) {
                 }));
     }
 
+
+    private boolean validateCredentials(String email, String password) {
+
+        // Reset errors.
+        emailWrapper.setError(null);
+        passwordWrapper.setError(null);
```

Comment:
why?

Reply:
set error method is used to set the error message. just when validate credentials is called it is initialised to null and then values are set

Reply:
Move this to presenter logic or ViewModel. Keep views free of logic

Reply:
Okay :+1: